runtime/jvmti/suspendhelper.cpp

```diff
@@ -42,7 +42,10 @@ suspendThread(J9VMThread *currentThread, jthread thread, BOOLEAN allowNull, BOOL
 		J9JavaVM *vm = currentThread->javaVM;
 #if JAVA_SPEC_VERSION >= 19
 		j9object_t threadObject = (NULL == thread) ? currentThread->threadObject : J9_JNI_UNWRAP_REFERENCE(thread);
-		if (NULL != targetThread)
+		/* If the mounted virtual thread is suspended then
+		 * the J9 PUBLIC FLAGS HALT THREAD JAVA SUSPEND flag will be set.
```

Comment:
- When the virtual thread is mounted, then the carrier thread is unmounted.
- When the virtual thread is unmounted, then the carrier thread is mounted.
```suggestion
		/* The J9 PUBLIC FLAGS HALT THREAD JAVA SUSPEND flag will be set
		 * if the thread is mounted.
		 */
```

Reply:
The comment statement is modified as suggested.